WORKSPACE

```diff
@@ -911,3 +911,71 @@ http_file(
         "https://storage.googleapis.com/builddeps/ab5a824d402c717bfe8e01cfb216a70fd4a7e1d66d2d7baa80ac6ad6581081c9",
     ],
 )
+
+http_file(
```

Comment:
How it is done? how one can add/remove files from this WROKSPACE. Is the list of images based on some kind of dependency analysis, or any command one can run?

Reply:
Someone pointed me to the ovirt-imageio rpms, so those are the imageio related ones. Then I discovered I needed the python3-systemd libraries installed, so found the RPMS for those and added.

Reply:
Note this is a manual process, I believe KubeVirt actually has process that can do some analysis, but haven't looked into it too much. But I can add the comment.

Reply:
If that is only needed for tests with fakeimageio, a comment would help

Reply:
I am trying to figure out why that one imageio test we have that is disabled is failing, maybe some of the things in here fix it. But the failure looks unrelated for now.